substrate/bin/node/runtime/src/lib.rs

```diff
@@ -2171,6 +2171,62 @@ impl pallet_broker::Config for Runtime {
 	type PriceAdapter = pallet_broker::CenterTargetPrice<Balance>;
 }
 
+
+parameter_types! {
+	// Id of the treasury
+	pub const PotId: PalletId = PalletId(*b"py/potid");
+
+	// Time needed after approval to unlock the reward claim
+	pub const Period:BlockNumber = 5*MINUTES;
+
+	// Maximum number of whitelisted projects
+	pub const MaxProjects:u32 = 50;
+
+	pub const EpochDurationBlocks: BlockNumber = EPOCH_DURATION_IN_BLOCKS;
+
+}
+impl pallet_distribution::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+
+	type NativeBalance = Balances;
+
+	/// Pot PalletId
+	type PotId = PotId;
+
+	/// A reason for placing a hold on funds.
+	type RuntimeHoldReason = RuntimeHoldReason;
+
+	/// This the minimum required time period between project whitelisting
+	/// and payment/reward_claim from the treasury.
+	type BufferPeriod = Period;
+
+	/// Maximum number of whitelisted projects
+	type MaxProjects = MaxProjects;
+
+	/// Epoch duration in blocks
+	type EpochDurationBlocks = EpochDurationBlocks;
+
+	type BlockNumberProvider = System;
+
+	type WeightInfo = pallet_distribution::weights::SubstrateWeight<Runtime>;
+}
+
+parameter_types!{	
+	pub const MaxWhitelistedProjects: u32 = 64;
+	pub const TemporaryRewards: Balance = 100000 * DOLLARS;
+	pub const TotalPeriod:BlockNumber = 30 * DAYS;
+	pub const LockPeriod:BlockNumber = 10 * DAYS;
+}
+impl pallet_opf::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type VoteLockingPeriod = LockPeriod;
+	type VotingPeriod = TotalPeriod;
+	type MaxWhitelistedProjects = MaxWhitelistedProjects;
+	type TemporaryRewards = TemporaryRewards;
+	type WeightInfo = pallet_opf::weights::SubstrateWeight<Runtime>;
+}
```

Comment:
This code block is a bit clumsy in terms of newline. Ofc compilers ignore newlines, but human eyes don't, and it kinda matters :) Please follow norms when it comes to formatting code.